src/endpoints/Server.php

```diff
@@ -34,20 +34,21 @@ public function __invoke(Application $app)
      */
     public function projects(Request $request, Response $response)
     {
+        // When using Directus in Docker (or any other service that relies on environment variables), always use `_` for
+        // the project key
         if (getenv("DIRECTUS_USE_ENV") === "1") {
-            $projectNames[]="_";
-        }
-        else {
+            $projectNames[] = "_";
+        } else {
             $basePath = \Directus\get_app_base_path();
             $scannedDirectory = \Directus\scan_folder($basePath.'/config');
 
             $projectNames = [];
-            if(empty($scannedDirectory)){
+            if (empty($scannedDirectory)) {
                 throw new NotInstalledException('This Directus instance has not been configured. Install via the Directus App (eg: /admin) or read more about configuration at: https://docs.directus.io/getting-started/installation.html#configure');
-            }else{
+            } else {
                 foreach($scannedDirectory as $fileName){
-                    if(!StringUtils::startsWith($fileName, 'private.')){
-                        $fileObject = explode(".",$fileName);
+                    if(!StringUtils::startsWith($fileName, 'private.') && !StringUtils::startsWith($fileName, '_')){
```

Comment:
What if the user registered `_` as their project? For that scenario; `projects` endpoint will return `[]` as a data.

Reply:
Good point. I guess we can update this with a check for length == 1